apps/backoffice-v2/src/lib/blocks/components/NodeBlock/NodeBlock.tsx

```diff
@@ -0,0 +1,4 @@
+import { INodeBlockProps } from '@/lib/blocks/components/NodeBlock/interfaces';
+import { FunctionComponent } from 'react';
+
+export const NodeBlock: FunctionComponent<INodeBlockProps> = ({ value }) => <>{value}</>;
```

Comment:
Don't know yet if this cell should have access to other props.
```suggestion
export const NodeCell: FunctionComponent<ExtractCellProps<'node'>> = ({ value }) => <>{value}</>;
```